### PR TITLE
update the randomness docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR updates the randomness docs by moving the lottery example to the tutorials section and creating a simple example on the precompile page for generating a random number

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.
- [x] If this page requires a disclaimer, I have added one

### After Translation Requirements

- [x] Will need to create PR in `moonbeam-docs` repo to remove images
- [x] Will need to create PR in `moonbeam-docs` repo to remove snippets

#### Items to be Updated

- Remove the `RandomnessLotteryDemo.sol` contract from the code snippets
- Replace the randomness images with the ones in the `new` directory

### CODE REVIEW

File: builders/pallets-precompiles/precompiles/randomness.md [hunk 1]
```diff
@@ -152,128 +152,161 @@ The happy path for a randomness request is shown in the following diagram:
...
+### Submit a Request to Generate a Random Number {: #request-randomness }
 
-1. Make sure you've switched to the account you want to participate with in MetaMask. You can verify the account that is connected under **ACCOUNT** 
-2. Enter the amount you want to contribute to the lottery in the **VALUE** field. It must be greater than or equal to the `PARTICIPATION_FEE` which is set to `100000 gwei` in the default contract
-3. Click on **participate**
-4. Confirm the transaction in MetaMask
+To request randomness, you're going to use the `requestRandomness` function of the contract, which will require you to submit a deposit as defined in the Randomness Precompile. You can submit the randomness request and pay the deposit by taking these steps:
 
-![Participate in the lottery](/images/builders/pallets-precompiles/precompiles/randomness/randomness-4.png)
+1. Enter an amount in the **VALUE** field for the fulfillment fee, it must be equal to or greater than the minimum fee specified in the `RandomNumber` contract, which is `500000` Gwei
```
Comment (albertov19) [inline builders/pallets-precompiles/precompiles/randomness.md:276]: Is `500000` a variable that we need to put in the variables file?

File: tutorials/eth-api/randomness-lottery.md [hunk 2]
```diff
@@ -0,0 +1,500 @@
...
+description: Looking to create a lottery smart contract? Follow this step-by-step tutorial on using Moonbeam's Randomness Precompile (a Solidity interface) to get started.
+---
+
+# Create a Lottery Contract using the Randomness Precompile
+
+![Randomness Moonbeam Banner](/images/builders/pallets-precompiles/precompiles/randomness/randomness-banner.png)
+
+_March 15, 2022 | by Erin Shaben_
+
+## Introduction {: #introduction }
```
Comment (albertov19) [inline tutorials/eth-api/randomness-lottery.md:12]: My main an only comment here is that probably adding a flow diagram of how the thing works may be helpful. I feel there is a LOT of text to go through

File: builders/pallets-precompiles/precompiles/randomness.md [hunk 3]
```diff
@@ -152,128 +152,161 @@ The happy path for a randomness request is shown in the following diagram:
...
-
-The lottery contract imports the `Randomness.sol` precompile and the `RandomnessConsumer.sol` interface, and inherits from the consumer contract. In the constructor of the contract, you can specify the source of randomness to be either local VRF or BABE epoch randomness.
-
-In general, the lottery contract includes functionality to check the status of the randomness request which will be used to determine whether the lottery is still accepting participants, if it has started, or if it has expired. It will use the `requestLocalVRFRandomWords` or `requestRelayBabeEpochRandomWords` function to select the random words, depending on which source of randomness you want to use. In addition, it will implement the `fulfillRandomWords` method and the callback will fulfill the request and use the random words to randomly pick the lottery winners.
+### Create a Random Number Generator Contract {: #create-random-generator-contract }
+
+The contract that will be created in this section includes the functions that you'll need at a bare minimum to request randomness and consume the results from fulfilling randomness requests.
```
Comment (albertov19) [inline builders/pallets-precompiles/precompiles/randomness.md:169]: Contract disclaimer? Educational purposes only?

File: tutorials/eth-api/randomness-lottery.md [hunk 4]
```diff
@@ -0,0 +1,500 @@
...
+
+### Compile & Deploy the Lottery Contract {: #compile-deploy-lottery-contract }
+
+To compile our contracts, you can simply run:
+
+```
+npx hardhat compile
+```
+
+![Hardhat Contract Compile]()
```
Comment (albertov19) [inline tutorials/eth-api/randomness-lottery.md:397]: Empty image 😜

**Review by albertov19 — CHANGES_REQUESTED** [review]
Left some feedback. Looks good! I liked the challenges

File: builders/pallets-precompiles/precompiles/randomness.md (hunk 1 above)
Comment (eshaben) [reply builders/pallets-precompiles/precompiles/randomness.md:278]: no, this is just for this example but users can test and configure this number

Comment (eshaben) [reply builders/pallets-precompiles/precompiles/randomness.md:281]: added a comment to the contract where the fulfillment fee is defined

**Review by albertov19 — APPROVED** [review]

**Pull request merged**